packages/charts/examples/components/demo/chart-types/area/area-chart-bi-directional-stacked/area-chart-bi-directional-stacked-example.component.ts

```diff
@@ -61,31 +67,47 @@ export class AreaChartBiDirectionalStackedExampleComponent implements OnInit {
         // This example demonstrates a scenario with time on the X scale and a numeric value on the Y scale.
         const scalesTop: IXYScales = {
             x: xScale,
-            y: new LinearScale(),
+            y: new LinearScale().reverse(),
         };
 
         const scalesBottom: IXYScales = {
             x: xScale,
-            y: new LinearScale().reverse(),
+            y: new LinearScale(),
```

Comment:
do we want to reverse these for the documentation example? it seems like the other way around would be the more typical orientation.